Core/Core/SwiftUIViews/WebView.swift

```diff
@@ -27,6 +27,8 @@ public struct WebView: UIViewRepresentable {
     private let source: Source?
     private var customUserAgentName: String?
     private var disableZoom: Bool = false
+    private var isPullToRefreshEnabled: Bool
+    private var pullToRefreshColor: UIColor?
```

Comment:
Does the color has any meaning if the pull to refresh is disabled? I'm thinking if we could use an enum like:
```
enum PullToRefresh {
    case disabled
    case enabled(color: UIColor)
}
```
We could change the parameter name from `isPullToRefreshEnabled` to just simply `pullToRefresh` so for example the initializer call would look like this: `WebView(url: myURL, pullToRefresh: .disabled)`

Reply:
That's a good suggestion. Color was just an afterthought that I added before opening the PR, but you are right. An enum would make it more clear.